Fix move mails shortcut in mailviewer

johnbotris opened the pull request:
There are two commits, the first is the basic bugfix, the second is an improvement in the UI, so that the dropdown doesn't open at the opposite side of the window to where the user would already be looking

We may choose to leave the second commit out for now, to avoid increasing the testing load

fix #4006

charlag reviewed (commented):
I do not understand why the bug did occur or why it got fixed.

I do not understand from the glance where the dropdown will be shown now

johnbotris commented:
> I do not understand why the bug did occur or why it got fixed.
> 
> I do not understand from the glance where the dropdown will be shown now

I updated the commit messages with more context and added a comment in `MailViewer._moveMails`

johnbotris commented on src/mail/view/MailViewer.ts:
```diff
@@ -1,5 +1,5 @@
 import {size} from "../../gui/size"
-import m, {Children, Component, Vnode} from "mithril"
+import m, {_NoLifecycle, Children, CommonAttributes, Component, Vnode, VnodeDOM} from "mithril"
```
it was leftover by accident. it's gone in the second commit anyway
